packages/flutter_migrate/README.md

```diff
@@ -0,0 +1,46 @@
+### Flutter Migrate
```

Comment:
Nit: The main heading should be a higher-level heading (e.g., `#` than the headings below it.